src/nimblepkg/options.nim

```diff
@@ -60,7 +61,9 @@ Usage: nimble COMMAND [opts]
 
 Commands:
   install      [pkgname, ...]     Installs a list of packages.
-               [-d, --depsOnly]   Install only dependencies.
+               [-D, --depsOnly]   Install only dependencies.
```

Comment:
This will surely break someone. Cannot change existing flags.

I presume you are trying to enable `-d:defineToPassNim` but does `-d:xyz` still get caught in the nimble code? Isn't it just looking for `-d`?

Reply:
yep, it will. i can add workaround for that.

Reply:
added workaround. now only `-d` is captured by nimble.